### PR TITLE
Fix name lookup example snap

### PR DESCRIPTION
**Pull request by hmalik88**
The current example snap is not inline with SIP-12, see https://github.com/MetaMask/snaps/pull/1768#discussion_r1334399505

### CODE REVIEW

File: packages/examples/packages/name-lookup/src/index.ts [hunk 1]
```diff
@@ -14,11 +13,11 @@ import { numberToHex } from '@metamask/utils';
 export const onNameLookup: OnNameLookupHandler = async (request) => {
   const { chainId, address, domain } = request;
 
+  // eslint-disable-next-line no-console
+  console.log(`Resolving for CAIP-2 chainId of: ${chainId}`);
```
Comment (FrederikBolding) [inline packages/examples/packages/name-lookup/src/index.ts:17]:
```suggestion
```

File: packages/examples/packages/name-lookup/src/index.ts [hunk 2]
```diff
@@ -14,11 +13,11 @@ import { numberToHex } from '@metamask/utils';
 export const onNameLookup: OnNameLookupHandler = async (request) => {
   const { chainId, address, domain } = request;
 
+  // eslint-disable-next-line no-console
```
Comment (FrederikBolding) [inline packages/examples/packages/name-lookup/src/index.ts:16]:
```suggestion
```

File: packages/examples/packages/name-lookup/src/index.ts [hunk 3]
```diff
@@ -14,11 +13,11 @@ import { numberToHex } from '@metamask/utils';
...
+  console.log(`Resolving for CAIP-2 chainId of: ${chainId}`);
+
   if (address) {
-    const shortAddress = `0x${address.substring(2, 5)}`;
-    const chainIdDecimal = parseInt(chainId.split(':')[1], 10);
-    const chainIdHex = numberToHex(chainIdDecimal);
-    const resolvedDomain = `example.domain - ${shortAddress} / ${chainIdHex}`;
+    const resolvedDomain = 'test.domain';
```
Comment (FrederikBolding) [inline packages/examples/packages/name-lookup/src/index.ts:20]: I agree with this change, but wanna check with @matthewwalsh0 how much this is gonna break on the petnames E2E side. Any thoughts?

Comment (FrederikBolding) [reply packages/examples/packages/name-lookup/src/index.ts:20]: I like that idea, what do you think @hmalik88 ?

File: packages/examples/packages/name-lookup/src/index.ts (hunk 1 above)
Comment (hmalik88) [reply packages/examples/packages/name-lookup/src/index.ts:17]: how come?

File: packages/examples/packages/name-lookup/src/index.ts (hunk 3 above)
Comment (hmalik88) [reply packages/examples/packages/name-lookup/src/index.ts:20]: Sure, that works.

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**